src/components/common/Auth.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
```

Comment:
```suggestion
import React, { useEffect, useState } from 'react';
```

Entiä ollenkaan halutaanko react-sovelluksessa importata reactia, mutta tää hiljensi lintterin

Reply:
Halutaan joo, mutta lintterit huomaa ton ajoittain hieman heikosti. Lisäsin